docs/organizations/roles-and-permissions-for-organizations.md

```diff
@@ -4,20 +4,74 @@ description: List of operations that users can perform on Codacy depending on th
+
+To review the additional permissions granted by the organization manager role, see the tables for each Git provider ([GitHub](#permissions-for-github), [GitLab](#permissions-for-gitlab), [Bitbucket](#permissions-for-bitbucket)).
+
+!!! important
+    Organization managers can access the **Policies** and **Integrations** settings sections of your organization and can therefore impact some repository settings for all repositories of your organization, even repositories that they can't access on the Git provider. However, they can't access the repositories themselves and can only see the repository names.
+
+### Assigning the organization manager role
+
+To assign the organization manager role:
+
+1.  Open your organization **Settings**, page **Roles and permissions**.
+
+1.  In the **Organization managers** area, use the search field to find the relevant organization member and click the member's name.
+
+    !!! note
+        You can only assign the organization manager role to [members of your organization](./managing-people.md#joining).
+
+    ![Security and risk management access management](images/roles-permissions-organization-manager-assign.png)
+
+### Revoking the organization manager role
+
+To revoke the organization manager role:
+
+1.  Open your organization **Settings**, page **Roles and permissions**.
+
+1.  In the **Organization managers** area, scroll the list to find the relevant user.
+
+1.  Click the **Revoke role icon** to the right of the user's name and confirm.
+
+## Configuring who can change the analysis configuration {: id="change-analysis-configuration"}
+
+By default, only users with the Codacy role **repository write** can change analysis configurations.
+
+To change this, open your organization **Settings**, page **Roles and permissions**, and define the lowest Codacy role required to perform the following operations on the repositories of your organization:
+
+-   [Ignore issues](../repositories/issues.md#ignoring-and-managing-issues)
+-   [Ignore files](../repositories-configure/ignoring-files.md)
+-   [Configure code patterns](../repositories-configure/configuring-code-patterns.md)
+-   [Configure file extensions](../repositories-configure/file-extensions.md)
+-   [Manage branches](../repositories-configure/managing-branches.md)
+-   [Reanalyze branches and pull requests](../faq/repositories/how-do-i-reanalyze-my-repository.md)
+
+![Configuring who can change analysis configurations](images/roles-permissions-repo-management.png)
+
+!!! note
+    Codacy determines the role of each organization member from the role of that member on your Git provider:
+
+    -   [GitHub](#permissions-for-github)
+    -   [GitLab](#permissions-for-gitlab)
+    -   [Bitbucket](#permissions-for-bitbucket)
+
 ## Permissions for GitHub
```

Comment:
At this point, the concept/task content above covers the Codacy **Roles and permissions** screen, it could make sense to separate this content from the three Git provider tables - reference content, which it's long enough to be on its own page.

What do you think of moving the three Git provider (reference) tables to a new page? Something like **Mapping Git provider roles**, or similar? 🤔

Reply:
As discussed, I'd still keep the entirety of the content on this page; it's still cohesive enough.

Eventually, we may separate once there's more content to add (e.g. billing managers or other new roles in the future).